src/components/UserLabel/README.md

```diff
@@ -89,19 +89,28 @@ LANDING_BLOCK-->
-| avatar       | User avatar                                 | [AvatarProps](../Avatar/README.md#properties) `string` `React.ReactElement` |              |
-| children     | Visible text                                |                              `React.ReactNode`                              |              |
 | view         | UserLabel view                              |                           `'outlined'` `'clear'`                            | `'outlined'` |
+| size         | UserLabel size                              |               `'3xs'` `'2xs'` `'xs'` `'s'` `'m'` `'l'` `'xl'`               |    `'s'`     |
+| avatar       | User avatar                                 | [AvatarProps](../Avatar/README.md#properties) `string` `React.ReactElement` |              |
+| text         | User text                                   |                              `React.ReactNode`                              |              |
+| description  | User description                            |                              `React.ReactNode`                              |              |
 | onClick      | `click` event handler for component itself  |                                 `Function`                                  |              |
 | onCloseClick | `click` event handler for button with cross |                                 `Function`                                  |              |
 | className    | Custom CSS class for root element           |                                  `string`                                   |              |
 | style        | HTML style attribute                        |                            `React.CSSProperties`                            |              |
 | qa           | HTML `data-qa` attribute, used in tests     |                                  `string`                                   |              |
-| size         | Avatar size                                 |                       `'xs'` `'s'` `'m'` `'l'` `'xl'`                       |    `'s'`     |
 
 ## CSS API
 
-| Name                         | Description      |
-| :--------------------------- | :--------------- |
-| `--g-user-label-font-size`   | Text font size   |
-| `--g-user-label-line-height` | Text line height |
+| Name                                     | Description                                             |
+| :--------------------------------------- | :------------------------------------------------------ |
+| `--g-user-label-size`                    | Size for avatar (width and height) and height for label |
+| `--g-user-label-border-radius`           | Label border radius                                     |
+| `--g-user-label-outer-gap`               | Label horizontal padding                                |
+| `--g-user-label-inner-gap`               | Gap between elements (avatar, text, close icon)         |
```

Comment:
These names are not meaningful enough. Can we improve them?

Reply:
Renamed to `--g-user-label-padding` and `--g-user-label-gap`